MiniDebug.cs

```diff
@@ -116,13 +116,16 @@ private set
     private readonly List<Renderer> _invRenders = new List<Renderer>();
     private Vector3 _noclipPos;
     public bool AcceptingInput { get; set; } = true;
+    private Vector3? cameraControllerPosition;
```

Comment:
Does this have to be nullable, since OnPreRenderCallback is (at least it'd better be) called before every OnPostRenderCallback?

Reply:
I did this because I only write to cameraControllerPosition if Camera Follow is true.

Reply:
Do you think putting a CameraFollow check into the post render callback would look cleaner?

Reply:
Would probably be fine, no idea what exactly the execution order of Update vs onPostRender are, because if Update is called before onPostRender there is a race condition.

Reply:
*Update before onPostRender and after onPreRender

Reply:
seems like it should be fine to check if CameraFollow is true, Update is called before OnPreRender and OnPostRender
![image](https://github.com/pseudorandomhk/MiniDebug/assets/25369813/6f6b6174-3186-46b3-a0a2-e92b3ceef38d)